saleor/site/migrations/0043_sitesettings_use_legacy_update_webhook_emission.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 5.2.8 on 2025-11-25 12:29
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("site", "0042_sitesettings_refund_reason_reference_type"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="sitesettings",
+            name="use_legacy_update_webhook_emission",
+            field=models.BooleanField(
+                db_default=True,
+                default=True,
```

Comment:
Also, this means, that any new Saleor instance created from scratch will use legacy webhook, which could cause that some one will use "old" approach by accident. I think that we should set this to `True` if there is any channel defined in DB, and set to False if databse is empty. WDYT?